app/policies/comment_policy.rb

```diff
@@ -0,0 +1,35 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def show?
+    user == @comment.owner ||
+    !comment.owner.private? ||
+    comment.owner.followers.include?(user)
+  end
+
+  def edit?
```

Comment:
The `edit` function is not necessary in this case since the `update` function is present.

Reply:
Don't you mean the opposite? The only way I can edit a comment is through the edit method not the update one.